plugins/ci/cmd/insights-ci/main.go

```diff
@@ -360,11 +342,104 @@ func getPlutoReport(configurationObject models.Configuration, manifestFolder str
 		Filename: "pluto.json",
 	}
 	// Scan with Pluto
-	plutoResults, err := ci.GetResultsFromCommand("pluto", "detect-files", "-d", manifestFolder, "-o", "json", "--ignore-deprecations", "--ignore-removals")
+	plutoResults, err := commands.Exec("pluto", "detect-files", "-d", manifestFolder, "-o", "json", "--ignore-deprecations", "--ignore-removals")
 	if err != nil {
 		return report, err
 	}
 	err = ioutil.WriteFile(configurationObject.Options.TempFolder+"/"+report.Filename, []byte(plutoResults), 0644)
+	if err != nil {
+		return report, err
+	}
 	report.Version = os.Getenv("plutoVersion")
 	return report, nil
 }
+
+func getConfiguration() (*models.Configuration, error) {
+	configFileName := "fairwinds-insights.yaml"
+
+	configFilePath := ""
+	basePath := ""
+	repoBasePath := ""
+
+	repoFullName := strings.TrimSpace(os.Getenv("REPOSITORY_NAME"))
```

Comment:
I'm using this env var to decide on normal vs cloned repo ctx, should we use something explicit?

Reply:
Yes I think something more explicit would be better, since both contexts have a repository name